SVProgressHUD/SVProgressHUD.m

```diff
@@ -249,7 +249,11 @@ + (void)showInfoWithStatus:(NSString*)status {
     
 #if TARGET_OS_IOS && __IPHONE_OS_VERSION_MAX_ALLOWED >= 100000
     dispatch_async(dispatch_get_main_queue(), ^{
-        [[self sharedView].hapticGenerator notificationOccurred:UINotificationFeedbackTypeWarning];
+        if (@available(iOS 10_0, *)) {
```

Comment:
I've never seen `@available` used with an underscore for the version number. Would you mind switching this to `if (@available(iOS 10.0, *)) {`?